src/frontend/apps/drive/src/features/explorer/components/modals/workspaces/ExplorerEditWorkspaceModal.tsx

```diff
@@ -0,0 +1,52 @@
+import {
```

Comment:
Do we really have to? Can't we eat both? I don't see the point of differentiating between creation and editing.